### PR TITLE
Replace CHANGELOG with GitHub's webview for releases

### PR DESCRIPTION
**Pull request by kennsippell**
This change replaces the Medic-Conf CHANGELOG with a link to the GitHub release view which shows commits grouped by tag.
#189 

Using `npm version minor` releases the need for custom scripts like `dirty-repo-check` since it makes similar assertions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -359,11 +359,9 @@ To compress PNGs and SVGs in the current directory and its subdirectories, two c
 
 # Releasing
 
-1. bump version number in `package.json`
-2. `npm install` to update `package-lock.json`
-3. `npm run generate-changelog` to update `CHANGELOG.md`
-4. `npm run release`
-5. try to work out how you can commit the CHANGELOG including the change that includes the CHANGELOG update before you run the release script
+1. Bump the version using `npm version minor` or `npm version major` as appropriate.
```
Comment (SCdF) [inline README.md]: You don't actually want to do this right? Because `npm run release` does this for you!

File: src/cli/usage.js [hunk 2]
```diff
@@ -52,7 +49,12 @@ ${bold('OPTIONS')}
 		Display application changelog.
 	
 	--accept-self-signed-certs
-		Allows medic-conf to work with self signed certs by telling node to ignore the error 
+    Allows medic-conf to work with self signed certs by telling node to ignore the error
+    
+${bold('SUPPORTED ACTIONS')}
+  Include a list of supported actions after '--':
```
Comment (SCdF) [inline src/cli/usage.js]: Is this true? I thought the supported actions were used *before* the `--`, and the `--` was used to signify that you wanted to pass params to an action you were invoking?

So like: `medic-conf do-a-thing -- some-param`

File: README.md (hunk 1 above)
Comment (kennsippell) [reply README.md]: I'm having the script run `npm version patch` by default. So we should still manually run `npm version minor` or `major` as appropriate.

File: src/cli/usage.js (hunk 2 above)
Comment (kennsippell) [reply src/cli/usage.js]: Definitely messed this up.

**Comment by kennsippell** [comment]
> If run in a git repo, it will also create a version commit and tag.
https://docs.npmjs.com/cli/version

So npm version will
* update the package.json and package-lock.json
* stage and commit that change
* set a tag

**Pull request merged**